web_src/css/explore.css

```diff
@@ -10,6 +10,29 @@
   margin-right: 5px;
 }
 
+.explore-lang .menu {
+  max-height: 60vh;
+  overflow-y: auto;
+}
+
+.explore-lang,
+.explore-sort {
+  float: right;
```

Comment:
If the page is using "flex" layout, why it needs "float: right" here?

Reply:
I think previous sort element had `float` as well, but we can likely replace this with flex as well.

Reply:
Will do a few CSS fixes here.

Reply:
Flex done in https://github.com/go-gitea/gitea/pull/24212/commits/1e59ce2bb4fdc0620c98ed6aebda89cbbcb2e994, no more float.

Reply:
> If the page is using "flex" layout, why it needs "float: right" here?

The parent element has display set to block.

I usually try to avoid nested flexboxes due to weird WebKit bugs which have been reported (for different projects) in the past.

would you like me change flex over float(will change if you prefer using flex instead)?